content/influxdb/cloud-iox/visualize-data/grafana.md

```diff
@@ -24,40 +25,62 @@ InfluxDB bucket backed by InfluxDB IOx.
 >
 > {{% caption %}}[Grafana documentation](https://grafana.com/docs/grafana/latest/introduction/){{% /caption %}}
 
-For the most performant queries, use SQL and the
-[Flight SQL protocol](https://arrow.apache.org/blog/2022/02/16/introducing-arrow-flight-sql/)
-to query InfluxDB.
+<!-- TOC -->
 
+- [Set up Grafana](#set-up-grafana)
+- [Download the Grafana Flight SQL Plugin](#download-the-grafana-flight-sql-plugin)
+- [Extract the Flight SQL plugin](#extract-the-flight-sql-plugin)
+- [Install the Grafana Flight SQL plugin](#install-the-grafana-flight-sql-plugin)
+    - [Install with Docker Run](#install-with-docker-run)
+    - [Install with Docker-Compose](#install-with-docker-compose)
+- [Configure the Flight SQL datasource](#configure-the-flight-sql-datasource)
+- [Query InfluxDB with Grafana](#query-influxdb-with-grafana)
+- [Build visualizations with Grafana](#build-visualizations-with-grafana)
 
-## Download the Flight SQL Plugin
+<!-- /TOC -->
+
+## Set up Grafana
+
+Follow the [Grafana.com instructions to Install Grafana](https://grafana.com/docs/grafana/latest/setup-grafana/installation/) for your system.
+
+{{% warn %}}
+Because Grafana Flight SQL Plugin is a custom plugin, you can't use it with Grafana Cloud.
+For more information, see [Find and Use Plugins in the Grafana Cloud documentation](https://grafana.com/docs/grafana-cloud/fundamentals/find-and-use-plugins/)
+{{% /warn %}}
```

Comment:
Move this down under "Download the Grafana Flight SQL plungin."

```suggestion
```

Reply:
> Move this down under "Download the Grafana Flight SQL plungin."

Thanks. Couldn't decide whether to keep both.